main/Main.hs

```diff
@@ -103,79 +138,152 @@ connectingSucceeded pool =
         putStrLn $ "Attempting to reconnect to the database in " <> (show delay::Text) <> " seconds..."
       return itShould
 
+{-|
+  This is where everything starts.
+-}
 main :: IO ()
 main = do
+  --
+  -- LineBuffering: the entire output buffer is flushed whenever a newline is 
+  -- output, the buffer overflows, a hFlush is issued or the handle is closed
+  --
+  -- no-buffering: output is written immediately and never stored in the buffer
```

Comment:
Maybe list as `NoBuffering` rather than `no-buffering` to match the camel case style of `LineBuffering` in this comment.